src/schemas/complaint-tool/schema.js

```diff
@@ -0,0 +1,171 @@
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+import _ from 'lodash';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'GI BILL SCHOOL COMPLAINT TOOL',
+  type: 'object',
+  additionalProperties: false,
+  definitions: {
+    fullName: _.merge({}, definitions.fullName, {
+      properties: {
+        prefix: {
+          type: 'string',
+          'enum': [
+            'Mr.',
+            'Mrs.',
+            'Ms.',
+            'Dr.',
+            'Other'
+          ]
+        }
+      }
+    })
+  },
+  required: [
```

Comment:
@lihanli how do we typically handle conditional requirements? For example name is only required if "anonymous" is not selected? Is this only handled on the FE? [Schema draft 7 includes if else support](https://ajv.js.org/keywords.html#ifthenelse), should we consider using that?

Reply:
something like this should work
```javascript
{
  oneOf: [
    {
      properties: {
        onBehalfOf: {
          type: 'string',
          'enum': [
            'Myself',
            'Someone else'
          ]
        },
      }
      required: ['fullName']
    },
    {
      properties: {
        onBehalfOf: {
          type: 'string',
          'enum': [
            'Anonymous'
          ]
        },
      }
    }
  ]
}
```

Reply:
Thanks @lihanli .